src/client/src/assets/subtract.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="12" height="3" viewBox="0 0 9 2" fill="none">
```

Comment:
Just curious, what was the thought process on using SVGs vs just using the characters `+` and `-`?

Reply:
Firstly, I used the + and - buttons rather than SVGs. However, the designs and styling resembled much more closely to the actual designs by the designer using these SVGs. 
Though, now I am facing a problem on how to change the color of SVGs based on the VSCode color theme :p

Reply:
Check this one out: https://css-tricks.com/cascading-svg-fill-color/